CODE_OF_CONDUCT.md

```diff
@@ -8,19 +8,19 @@ In the interest of fostering an open and welcoming environment, we as contributo
 
 Examples of behavior that contributes to creating a positive environment include:
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+-   Using welcoming and inclusive language
```

Comment:
These changes are a part of the pre-commit hook.

Reply:
No problem. I'd added the Code of Conduct via GitHub's interface, and so it wasn't run through Prettier.